ext/storage_sources/local_store/local_store.c

```diff
@@ -467,11 +467,13 @@ local_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *sessi
 
 err:
     if (ret != 0) {
```

Comment:
What about switching the conditional? I was about to suggest combining with an `&&` and wonder if someone in the future would mistakenly do that. What about:
```
if (ret == 0)
    *file_systemp =...
else if (fs != NULL) {
    free(...)
}
return (ret);
```

Reply:
I like it.  Thanks!